content/kubernetes/release-notes/k8s-6-2-18-3.md

```diff
@@ -0,0 +1,144 @@
+    /kubernetes/release-notes/k8s-6-2-18-3.md,
+    /kubernetes/release-notes/k8s-6-2-18-3/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.2.18-3 supports the Redis Enterprise Software release 6.2.18 and includes feature improvements and bug fixes.
+
+The key bug fixes, new features, and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.18-58` or  `redislabs/redis:6.2.18-58.rhel8-openshift`
+* **Operator**: `redislabs/operator:6.2.18-tbd`
+* **Services Rigger**: `redislabs/k8s-controller:6.2.18-tbd` or `redislabs/services-manager:6.2.18-tbd` (on the Red Hat registry)
+
+## New features
+
+* Support for [Redis on Flash]({{<relref "/kubernetes/re-clusters/redis-on-flash.md">}}) (RoF) (RED-78613)
+
+## Feature improvements
+
+* The `podSecurityPolicyName` field in `RedisEnterpriseCluster` resources is now deprecated for Kubernetes versions 1.24 or earlier, and invalid for 1.25 and later. Customers are advised to switch to using the [`PodSecurityAdmission`](https://kubernetes.io/docs/concepts/security/pod-security-admission/) or alternative methods to enforce pod security (RED-81921).
+
+* Added support for VMware [Tanzu Kubernetes Grid (TKG)](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html), in addition to [Tanzu Kubernetes Grid Integration Edition (TKGI)](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid-Integrated-Edition/index.html) that was previously and is still supported (RED-65630).
+
+* Added support for PEM encryption through the [Redis Enterprise cluster API](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md)(RED-78613).
+
+* Hardened security context constraints to align with standards for OpenShift 4.11 (RED-83215).
+
+* Changed log collector default to avoid collection of non-Redis Enterprise logs and items (RED-83216).
+
+* Allowed configuration of the Redis Enterprise cluster (REC) service type (RED-84644).
+
+## Bug fixes
+
+* Allow any ingress class name annotation when using NGINX ingress controller. This is no longer required to be exactly `nginx` (RED-79205).
+* Fixed log collector handling of namespace parameter on Windows (RED-83532).
+* Fixed issue with updating credentials on Openshift when accessing the cluster externally with routes (RED-73251, RED-75329).
+
+## API changes
+
+The following fields were added to the [Redis Enterprise cluster (REC) API](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md):
+
+* Added `.services.apiService.type` to allow configuration of the API service type.
+* Made `.redisOnFlashSpec` available by default.
+* Made `.ocspConfiguration` available by default for configuring OCSP stapling.
+* Made `encryptPkeys` available by default for configuring PEM encryption.
+
+The following fields were added to the [Redis Enterprise database (REDB) API:
```

Comment:
Did you want to link the REDB API doc here?
If not, let's remove the `[` character.